instance/tests/integration/integration_instance.py

```diff
@@ -362,7 +361,9 @@ def test_external_databases(self):
         if not settings.DEFAULT_INSTANCE_MYSQL_URL or not settings.DEFAULT_INSTANCE_MONGO_URL:
             print('External databases not configured, skipping integration test')
             return
-        OpenEdXInstanceFactory(name='Integration - test_external_databases')
+        OpenEdXInstanceFactory(
```

Comment:
@giovannicimolin , can you revert this and the similar changes to formatting now that we do not require an additional argument to be passed?

Reply:
Done!